tweepy/pagination.py

```diff
@@ -16,6 +15,12 @@ class Paginator:
 
     .. versionadded:: 4.0
 
+    .. note::
+
+        When passing ``return_type=requests.Response`` to :class:`Client` for
+        pagination, payload of response will be deserialized implicitly to get
+        ``meta`` attribute every requests, which may affect performance.
```

Comment:
```suggestion
        When the returned response from the method being passed is of type
        :class:`requests.Response`, it will be deserialized in order to parse
        the pagination tokens, likely negating any potential performance
        benefits from using a :class:`requests.Response` return type.
```